(2) Vox Populi/Database Changes/Text/en_US/Civilizations/NewCivilizationText.xml

```diff
@@ -104,10 +104,10 @@
 			<Text>A cothon is an artificial, protected inner harbor such as that in Carthage during the Punic Wars around 200 BC. The cothon at Carthage was divided into a rectangular merchant harbor followed by an inner protected harbor reserved for military use only. This inner harbor was circular and surrounded by an outer ring of structures divided into a series of docking bays for ship maintenance, along with an island structure at its centre that also housed navy ships. Each individual docking bay featured a raised slipway. Above the raised docking bays was a second level consisting of warehouses where oars and rigging were kept along with supplies such as wood and canvas. On the island structure there existed a raised 'cabin' where the admiral in command could observe the whole harbor along with the surrounding sea. Altogether the inner docking complex could house up to 220 ships. The entire harbor was protected by an outer wall and the main entrance could be closed off with iron chains.</Text>
 		</Row>
 		<Row Tag="TXT_KEY_BUILDING_GREAT_COTHON_STRATEGY">
-			<Text>Unique Carthaginian replacement for the {TXT_KEY_BUILDING_EAST_INDIA}. In addition to the bonuses of the {TXT_KEY_BUILDING_EAST_INDIA}, the {TXT_KEY_BUILDING_GREAT_COTHON} provides two [ICON_INTERNATIONAL_TRADE] Trade Routes, more poverty reduction, more [ICON_GOLD] Gold for both recipient and target of trade routes from the City where it is built; and boosts all Lighthouses and Harbors in your empire. Unlike the {TXT_KEY_BUILDING_EAST_INDIA}, the {TXT_KEY_BUILDING_GREAT_COTHON} is made available in the [COLOR_CYAN]Classical Era[ENDCOLOR] with the discovery of [COLOR_CYAN]{TXT_KEY_TECH_CURRENCY_TITLE}[ENDCOLOR], and requires a {TXT_KEY_BUILDING_MARKET} in the city instead of a {TXT_KEY_BUILDING_MINT}.</Text>
+			<Text>Unique Carthaginian replacement for the {TXT_KEY_BUILDING_EAST_INDIA}. In addition to the bonuses of the {TXT_KEY_BUILDING_EAST_INDIA}, the {TXT_KEY_BUILDING_GREAT_COTHON} provides a free {TXT_KEY_BUILDING_HARBOR} in all of your coastal cities, two [ICON_INTERNATIONAL_TRADE] Trade Routes, more poverty reduction, more [ICON_GOLD] Gold for both recipient and target of trade routes from the City where it is built; and boosts all Lighthouses and Harbors in your empire. Unlike the {TXT_KEY_BUILDING_EAST_INDIA}, the {TXT_KEY_BUILDING_GREAT_COTHON} is made available in the [COLOR_CYAN]Classical Era[ENDCOLOR] with the discovery of [COLOR_CYAN]{TXT_KEY_TECH_CURRENCY_TITLE}[ENDCOLOR], and can only be built in a Coastal City.</Text>
```

Comment:
Should add the "no building requirement", since that's a feature over the base NW.